code/obj/flock/cage.dm

```diff
@@ -12,6 +12,7 @@
 	health_max = 30
 	alpha = 192
 	anchored = FALSE
+	hitTwitch = FALSE
```

Comment:
I think a clarifying comment saying that it implements its own twitch stuff would be nice